packages/react-button/etc/react-button.api.md

```diff
@@ -78,25 +78,26 @@ export interface ButtonState extends ButtonProps {
 
 // @public (undocumented)
 export type ButtonTokens = ColorTokens & FontTokens & {
-    paddingLeft?: string;
-    paddingRight?: string;
-    paddingTop?: string;
-    paddingBottom?: string;
+    paddingLeft?: string | number;

```

Comment:
I don't think we should allow for number. Or at least not yet. 

When you use `useInlineTokens`, it will not auto expand numbers into PX. We could add it, it will cost a bit in perf, and it should be done right. Until that works, i'd suggest we stick with strings.

Reply:
Removed!